src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -5,21 +5,25 @@ import React from "react";
  *  Breadcrumb component
  */
 export function Breadcrumb(props) {
+  const Component = props.component || "a";
   return (
-    <nav className="ds-pt-4 ds-pb-4" aria-label="breadcrumbs" id={props.id}>
+    <nav className="ds-py-6" aria-label="breadcrumbs" id={props.id}>
       <ul className="ds-block ds-text-custom-blue-dark ds-leading-23px ds-font-body">
         {props.items
           ? props.items.map((item, key) => {
               return (
-                <li key={key * key} className={`ds-inline-block ds-w-100`}>
-                  <a
+                <li
+                  key={key * key}
```

Comment:
Why is this `key` value complex? If there's a unique identifier inside `props.items`, can we do something like:
`key={'items-${item.id}'}` or `key={'items-${index}'}` (Apostrophes instead of backticks)

Reply:
It was like this before, there is no id in `index`, I updated to use index as key